docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+   AlexNet model. You are basically done at this point, and your final package
+   should look similiar to the picture below. Remember, the actual model is now contained
+   in the ``models/base_models.py`` file, and that is what is getting run on our site to get a score for you.
+    .. image:: /docs/source/modules/tutorial_screenshots/final_submit.png
+      :width: 600
+8. You are now ready to submit! Zip the folder named ``my_alexnet_subission``,
+   navigate to http://www.brain-score.org/profile/, log in/create a new account,
+   and submit the model! Usually (depending on how busy the time of year is)
+   it will take around 1 hour or so to score, but might take longer. If you
+   do not see a score within 24 hours, contact us and we can send you
+   (soon you will have access to this yourself)
+   the error logs to resubmit. You have now successfully submitted a model!
+   Congrats, and we look forward to having more submissions from you.
+   In the future, you can just copy the submission package and paste
+   in your code into ``models/base_models.py``, and it should work (which
+   is why we had you make that whole package in the first place!)
+9. *Experiment Further*: Try submitting different pretrained models,
+   like ``vgg16``, ``Inception``, or ``RESNET``, that are available from
+   Pytorch. Keras works the same way, and for the most part this
+   tutorial can be followed to use ``keras.py`` instead of ``pytorch.py``; if you
+   use Keras, just remember to add it to the ``setup.py`` file. Finally,
+   when using a different pretrained model (or a custom model) see the
+   section below on layer selection to get the layers needed in the actual model,
+   which will be different according to the model used.
+
+
+
+Submitting a Model to Brain-Score.org Part 3: Custom model (Optional)
+=====================================================================
+
+At this point, I would say that you are pretty comfortable with the submission,
+and hopefully you have submitted at least one model and gotten a score.
+So, in this section, I will skip some of the parts that are common with
+submitting a custom model (vs. something like AlexNet), and just focus on what is different.
+
+1. In short, submitting a custom model is not that difficult
+   for those that have already submitted a model like AlexNet
+   and have a submission package ready. If you have not done this,
+   I highly recommend going through this tutorial beforehand, or else you will
+   encounter some errors along the way.
+2. The entire package we submit will be the same as a pretrained model,
+   but with the ``models/base_models.py`` file different (as the model itself is different).
+   So, I would recommend just copying the ``my_alexnet_submission`` folder,
+   pasting it into the ``my_model_submissions`` folder, and renaming it to something
+   like ``my_custom_submission``. This will take care of all the tricky
```

Comment:
like above, just start directly from the directory structure given by `sample-model-submission`

Reply:
same ping as above